staging/kos/README.md

```diff
@@ -139,6 +142,25 @@ The SDN has just four concepts, as follows.
 A controller chooses the IP address for each NetworkAttachment.  An
 attachment's MAC address is a function of the IP address and the VNI.
 
+Neither controller enforces any interlock between its actions and the
+lifecycle of any API object. Consequently, there can not be any fully
+effective enforcement of immutability of any field of any API object ---
+the client can always delete and object and create a replacement with the
+same name, possibly without the controller seeing any intermediate state.
+Coping with all possible changes makes for complicated controller code.
+Stay tuned for lifecycle interlocks.
```

Comment:
Perhaps we should be more explicit than "stay tuned".  How about something like "A future PR may contribute an alternative that uses such interlocks, illustrating their value and implementation issues" ?

Reply:
Updated with words similar to those you suggested in otherwise unrelated PR #69.